sql/lessons_summary.txt

```diff
@@ -0,0 +1,128 @@
+SQL student path
```

Comment:
I don't quite get what this file is for? It's not referred to anywhere.. did you accidentally add your lesson plan to the PR? :D

Reply:
It should not be in the PR. It was meant to be a guide while writing and I accidentally added it to the PR.
I'll remove it